optical_flow:
    ros__parameters:
        timer_period: 0.01 # min 0.01
        sensor_timeout: 0.5
        parent_frame: odom
        child_frame: base_link
        x_init: 0.0 # Initial x position in meters
        y_init: 0.0 # Initial y position in meters
        z_height: 0.025 # Height of the sensor from the base frame in meters
        board: paa5100 # [pmw3901, paa5100]
        motion_scale: 5 # Scale factor for motion calculation
        spi_nr: 0
        spi_slot: front # [front, back]
        rotation: 0 # [0, 90, 180, 270]
        publish_tf: true
